### PR TITLE
Moving the set UTF-8 and binary modes out of the login method

### PR DESCRIPTION
**Pull request by warrenbailey**
Fix for #89

### CODE REVIEW

**Comment by jlaffaye** [comment]
I agree to move this code out of the login function.

I would rather Binary() function be a SetTransfertType with takes the type as an argument.
That type would be an enum with values from http://www.nsftools.com/tips/RawFTP.htm#TYPE

File: ftp.go [hunk 1]
```diff
@@ -128,17 +128,14 @@ func (c *ServerConn) Login(user, password string) error {
...
-		return err
-	}
-
-	// Switch to UTF-8
-	if err := c.setUTF8(); err != nil {
+	if _, _, err := c.cmd(StatusCommandOK, "TYPE I"); err != nil {
 		return err
 	}
-
 	return nil
```
Comment (corny) [inline ftp.go]: Shorter:
```go
_, _, err := c.cmd(StatusCommandOK, "TYPE I")
return err
```

**Comment by mailtruck** [comment]
Hey @jlaffaye can you please give a little more guidance on what you would like to see here?

Where would we store the actual mappings for the transfer types? Saved as constants?

What would the actual type of the `transfer type` argument be? Would we define our own string type?

P.S. Thank you for this library!

**Comment by warrenbailey** [comment]
Tidying up Pull requests, no longer needed

**Pull request closed without merging**